test/unit/pdf_find_controller_spec.js

```diff
@@ -21,6 +21,10 @@ import { PDFFindController } from "../../web/pdf_find_controller.js";
 import { SimpleLinkService } from "../../web/pdf_link_service.js";
 
 const tracemonkeyFileName = "tracemonkey.pdf";
+const CMAP_PARAMS = {
+  cMapUrl: isNodeJS ? "./external/bcmaps/" : "../../../external/bcmaps/",
+  cMapPacked: true,
+};
```

Comment:
Please import this from https://github.com/mozilla/pdf.js/blob/master/test/unit/test_utils.js instead.

Reply:
Yep but the path in `test_utils.js` is not correct here (note the `../../..`), it's why I recreate one here.